src/transitions/Slide.spec.js

```diff
@@ -181,4 +182,37 @@ describe('<Slide />', () => {
       });
     });
   });
+
+  describe('slidemount', () => {
+    let mount;
+    let mountWrapper;
+    let instance;
+
+    before(() => {
+      mount = createMount();
+      mountWrapper = mount(<Slide><div /></Slide>);
+    });
+
+    after(() => {
+      mount.cleanUp();
+    });
+
+    it('default value', () => {
+      mountWrapper.setProps({ in: false });
+      instance = mountWrapper.instance();
+      instance.componentDidMount();
+      assert.strictEqual(
+        ReactDOM.findDOMNode(mountWrapper.instance().transition).style.transform,
```

Comment:
Otherwise, we could be using `if (X instanceof HTMLElement)` but that might fails with some old browser through browserstack.